dev/user.clj

```diff
@@ -15,9 +15,10 @@
    [clojure.string :as string]
    [com.grzm.component.pedestal :as pedestal-component]
    [com.stuartsierra.component :as component]
-   [com.stuartsierra.component.repl :refer [reset set-init]]
+   [com.stuartsierra.component.repl :refer [reset set-init system start stop]]
```

Comment:
Generally, never use `start` or `stop`. Better remove them from here so you're not tempted.

`reset` is idempotent, particularly the 'rehardened' snippet which you should always use

Reply:
I understood that from you before. :slightly_smiling_face: